packages/stack-deploy-iam/bin/app.ts

```diff
@@ -52,6 +54,18 @@ class StackDeployUser extends cdk.Stack {
                })
           );
 
+         /**
+          * See if custom policy statements are provided
+          *  and custom policy statements are directly attached to the group policy
+          */
+         if (CUSTOM_POLICY_PATHS) {
+             console.log("Custom policy statement(s) has been provided")
```

Comment:
We generally don't log anything in our cdk stacks (but maybe we should?)

Reply:
In this case, I just wanted to acknowledge that we are injecting a policy, considering this as a CLI application :)